### ISSUE TITLE
label on form in index.html

### ISSUE THREAD

**Issue by tspeed90** [fac-13/phac-kpnet#63]
I believe your labels should be inside of your form surrounding your input. Also, you aren't using labels, just including the tag. If you want to label the field for accessibility but don't want to display a visible label of "search" or whatever, you can use aria-labels on the input.
